trojsten/contests/models.py

```diff
@@ -322,6 +324,12 @@ class Task(models.Model):
         max_length=128, verbose_name='Odkaz na externé odovzdávanie',
         blank=True, null=True,
     )
+    email_on_desc_submit = models.BooleanField(
+        verbose_name='Zaslať notifikáciu o submite popisu opravovateľom', default=False
+    )
+    email_on_code_submit = models.BooleanField(
+        verbose_name='Zaslať notifikáciu o submite kódu opravovateľom', default=False
```

Comment:
to isté

Reply:
Ešte tieto popisy by si mal preložiť, inak to je už za mňa OK. Neviem, či @maaario  má ešte niečo?

Reply:
Prekladom nic nepokazis. Eventualne chceme mat prelozene vsetky user facing veci (idealne aj v adminovi). Navyse tieto popisky robia problemy pri py2 vs py3 migraciach, kedze sa generuju rozdielne.